examples/resources/audio/classic.go

```diff
@@ -1,4 +1,5 @@
 // Code generated by file2byteslice. DO NOT EDIT.
+// (gofmt is fine after generating)
```

Comment:
I think you are using an old version of file2byteslice.

Reply:
My bad

Reply:
I updated it and it still stays the same
:/

Reply:
Fixed